docs/advanced-usage.md

```diff
@@ -8,6 +8,7 @@ the build will not fail, which is very useful for legacy projects.
  * [Improve the report with a base URL](#improve-the-report-with-a-base-URL)
  * [Add exclusions with `exclude` filters](#add-exclusions-with-exclude-filters)
  * [Add exclusions with Android build variants](#add-exclusions-with-android-build-variants)
+ * [Consume rules from artefact](#consume-rules-from-artefact)
```

Comment:
isn't [`artifact` though](https://stackoverflow.com/questions/2487485/what-is-a-maven-artifact)?

Reply:
Both spellings go:

![image](https://user-images.githubusercontent.com/153802/36663754-e5b9f25e-1ae2-11e8-988e-e442d2d862db.png)